nifi-nar-bundles/nifi-email-bundle/nifi-email-processors/src/main/java/org/apache/nifi/processors/email/ListenSMTP.java

```diff
@@ -135,29 +89,17 @@
             .addValidator(StandardValidators.TIME_PERIOD_VALIDATOR)
             .build();
 
-    protected static final PropertyDescriptor SMTP_MAXIMUM_MSG_SIZE = new PropertyDescriptor.Builder()
+    static final PropertyDescriptor SMTP_MAXIMUM_MSG_SIZE = new PropertyDescriptor.Builder()
```

Comment:
@olegz I see where you come from but put file is controlled by the DFM, ListenSMTP is an exposed service

True, we don't do it everywhere but it is present on GetFile as well as other processors crossing administrative domains (DFM <> FS) and so it goes